_posts/2017-10-17-meetup.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How to Start a JavaScript Synthpop Band
+author: lon
+layout: post
+categories:
+- posts
+- meetups
+when: 2017-10-17T19:30:00-05:00
+---
+
+{% assign speakr = 'J. Kyle Fagan' %}
+{% assign handle = 'jkyle' %}
+{% assign twiturl = 'https://twitter.com/jkyle' %}
+
+Synthesizers! JavaScript! Aleatoricism?? Using an actual analog synthesizer as a
+guide, Kyle will build a feature-for-feature replica in the browser with the Web
+Audio API and show how to use JavaScript to overcome a lack of musical training.
+You'll learn the capabilities and limitations of the Web Audio API, and how it
+can be used for applications like gaming, WebVR, and 80s retro synthwave. With
+live coding, live synth playing, and a touch of music theory, <em>what could go wrong?</em>
+
+<h3>Our speaker</h3>
+
+<div class="media-object speaker-bio">
+  <a href="{{twiturl}}">
+    <img alt="{{speakr}} @jkyle on Twitter"
+      src="https://avatars1.githubusercontent.com/u/100188?v=4&s=460" />
+  </a>
+  <div>
+  <a href="{{twiturl}}"><strong>{{speakr}}</strong></a>
+  is an Austin local and writes javascript for <a href="https://www.eduphoria.net">Eduphoria</a>.
+  He's an aspirational artist, aspirational musician, and aspirational cool dad.
+  </div>
+</div>
+
+Not only did [Spredfast][] fly Ben out, but they are providing food and drink,
+so be sure to tell them thanks!
```

Comment:
s/👆//

Reply:
It's what I get for copy/paste.

Reply:
How about this:

> Make sure to thank our gracious hosts [Spredfast][], and our sponsors
 +[eRelevance][]!

And add this at bottom:
> [eRelevance]: http://www.erelevancecorp.com/careers/